usaspending_api/disaster/tests/fixtures/federal_account_data.py

```diff
@@ -162,6 +162,8 @@ def generic_account_data():
         disaster_emergency_fund=defc_m,
         treasury_account=tre_acct1,
         distinct_award_key="0wefjwe|3443r||",
+        ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe=0,
```

Comment:
Without non-zero values in mock data, are the tests really testing the new logic?

Reply:
I created new mock data for the outlay tests just to make the changes more obvious for those tests, I can go back and add data and alter the existing tests if you prefer.

Reply:
Ah, I overlooked the intentions of the new tests. When I was working on the hotfix changes it was pretty easy to alter the mock data values to still equal the original result values. Fine if you want to leave the tests as-is